### DISCUSSION TITLE
Question about AWS RDS and STS versions

### DISCUSSION THREAD

**Discussion by pcalouche** [aws/aws-advanced-jdbc-wrapper#1166] [General]
I noticed in your releases there are different versions of RDS and STS as transitive dependencies.  For instance, in the 2.4.0 release `rds` is version `2.25.70` and the `sts` version is `2.27.22`.  Why do you not keep these at the same version given that is how they are managed in the AWS V2 SDK bom?  See https://mvnrepository.com/artifact/software.amazon.awssdk/bom/2.25.70

**Comment by karenc-bq** [comment]
Hi @pcalouche, thank you for raising this. Our dependencies are managed by Dependabot so they may be updated at different rates, we will manually update them to make sure the versions align.

**Comment by pcalouche** [comment]
Thank you!
